lib/aws.js

```diff
@@ -35,54 +35,46 @@ class InstanceFetcher {
     }
 
     onInstances(result) {
```

Comment:
`onInstances` can probably just be folded into `getInstances` at this point since it isn't called anywhere else